### PR TITLE
chore(edit-page-2): Add Functionality for Workflow Actions in Edit Content

### LINKED ISSUE #27549: Edit Page V2: Enable `Move` workflow action in Edit Content inside the Edit Page
### Parent

https://github.com/dotCMS/core/issues/27546

### User Story
As a content editor, I want to be able to use the move button in edit mode, so I can easily move content to a different location within the dotCMS system.

```[tasklist]
### Acceptance Criteria
- [x] The edit contentlet dialog: "Move" action should show the dialog
- [x] Include comment and assign.
```

### Proposed Objective
Core Features

### Proposed Priority
Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
N/A

### Assumptions & Initiation Needs
- The move button is already present in the user interface, but is currently not functional.
- The move operation should be available for all types of content in dotCMS.

### Quality Assurance Notes & Workarounds
N/A

### PR DESCRIPTION
**Pull request by zJaaal**
## Proposed changes
* Create new WorkflowActions service based in the old one to work with the new dialog.

## Screenshot

https://github.com/dotCMS/core/assets/63567962/12e6d4d3-1d99-4641-ae99-94190b8330a8

### CODE REVIEW

File: core-web/libs/ui/src/lib/modules/dot-dialog/dot-dialog.component.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
         position: absolute;
         right: 0;
         top: 0;
-        z-index: 1000;
+        z-index: 1103;
```
Comment (KevinDavilaDotCMS) [inline core-web/libs/ui/src/lib/modules/dot-dialog/dot-dialog.component.scss:17]: mmm maybe this change break a confirm dialogs component
Have you tried (if is needed)  to open a confirmation dialog on this component?

Comment (zJaaal) [reply core-web/libs/ui/src/lib/modules/dot-dialog/dot-dialog.component.scss:17]: We need to test this when we fix this https://github.com/dotCMS/core/issues/28177

File: core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.ts [hunk 2]
```diff
@@ -177,13 +189,66 @@ export class DotEmaDialogComponent {
...
+    /**
+     * Handle workflow event
+     *
+     * @param {DotCMSWorkflowActionEvent} event
+     * @memberof DotEmaDialogComponent
+     */
+    handleWorkflowEvent(event: DotCMSWorkflowActionEvent) {
+        this.workflowActions
+            .handleWorkflowAction(event, this.callEmbeddedFunction.bind(this))
+            .subscribe(({ callback, args, summary, detail }) => {
```
Comment (fmontes) [inline core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.ts:210]: take1?

File: core-web/libs/portlets/edit-ema/portlet/src/lib/services/dot-ema-workflow-actions/dot-ema-workflow-actions.service.ts [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+        return this.dotWizardService.open<DotWorkflowPayload>(wizardInput).pipe(
+            filter(() => !!wizardInput),
+            take(1)
+        );
+    }
+
+    fireBulkWorkflowAction(
+        event: DotCMSWorkflowActionEvent,
+        data?: DotWorkflowPayload
+    ): Observable<DotActionBulkResult | MessageInfo> {
```
Comment (fmontes) [inline core-web/libs/portlets/edit-ema/portlet/src/lib/services/dot-ema-workflow-actions/dot-ema-workflow-actions.service.ts:226]:
```suggestion
    }

    openWizard(event: DotCMSWorkflowActionEvent): Observable<DotWorkflowPayload> {
        const wizardInput = this.setWizardInput(
            event.workflow,
            this.dotMessageService.get('Workflow-Action')
        );

        return this.dotWizardService.open<DotWorkflowPayload>(wizardInput).pipe(
            filter(() => !!wizardInput),
            take(1)
        );
    }

    fireBulkWorkflowAction(
        event: DotCMSWorkflowActionEvent,
        data?: DotWorkflowPayload
    ): Observable<DotActionBulkResult | MessageInfo> {
```

Also doc?

**Review by fmontes — APPROVED** [review]

**Review by KevinDavilaDotCMS — APPROVED** [review]

**Pull request merged**